components/contribution-flow/FeesOnTopInput.js

```diff
@@ -27,11 +26,7 @@ const getOptionFromPercentage = (amount, currency, percentage) => {
     // Value must be unique, so we set a special key if feeAmount is 0
     value: feeAmount || `${percentage}%`,
     percentage,
-    label: (
-      <span>
-        {formatCurrency(feeAmount, currency)} <Span color="black.500">({percentage * 100}%)</Span>
```

Comment:
I assume that a string was needed as an output, if that's the case could we rather put that as `aria-title` (or similar, I'm not sure which one to use) to keep the formatting?

Also relying on the standard `formatCurrency` is a good practice, would be great if we can keep it

Reply:
Done. 👍🏽